project1.css

```diff
@@ -29,17 +31,45 @@ section {
   top: 92px;
 }
 
+@media screen and (min-width: 480px) {
+  section {
+    display: flex;
+    flex-direction: column;
+    align-items: flex-start;
+    padding: 114px 24px;
+    gap: 12px;
+    position: absolute;
```

Comment:
- [ ] I would strongly recommend that you should not use `position: absolute` to layout your section contents because absolute positioning removes the positioned element from the natural flow of the document which can cause maintenance problems and your site will not be responsive. You can consider using `flex or grid `